scheduler/clients/javascript/lib/baseClient.ts

```diff
@@ -77,57 +71,58 @@ export class APIResponse<T> {
   }
 }
 
-/**
- * Credentials for the API for end users (usually frontend)
- */
-export class UserCreds implements ICredentials {
-  private readonly nettuAccount: string
-  private readonly token?: string
-
-  constructor(nettuAccount: string, token?: string) {
-    this.nettuAccount = nettuAccount
-    this.token = token
+export const createAxiosInstanceFrontend = (
```

Comment:
Can you document? What's the difference between this and `createAxiosInstanceBackend`?

Reply:
Updated 👍 

The main difference is that one allow `keepAlive`, which needs the function to be `async` as it needs to import a Node package.

Technically, it could be only one function, but I think it makes it easier for the frontend to still have a function that isn't async. That way, it can be defined directly as a variable in a file (`export ....`) on the frontend